beamer/tests/config/util.py

```diff
@@ -18,6 +24,25 @@ def write_keystore_file(path, private_key, password):
     path.write_text(json.dumps(obj))
 
 
+def read_config_state(rpc_file, artifact):
+    root = pathlib.Path(__file__).parents[3]
```

Comment:
I foresee some time of debugging if we ever change the directory structure here 🤓

Reply:
> I foresee some time of debugging if we ever change the directory structure here nerd_face

Hm, yeah, that's not too ideal and we already have a couple, including one in `test_cli` from agent tests.
So I added `beamer.tests.util.get_repo_root` and updated all the places to make use of that function instead of doing it manually.